### PR TITLE
Updating VLF_Singlet

### PR DESCRIPTION
**Pull request by leonidasprado**
Still not sure about xqcut and qcut

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat [hunk 1]
```diff
@@ -49,7 +49,8 @@
 # PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
 #*********************************************************************
  lhapdf = pdlabel      ! PDF set
```
Comment (leonidasprado) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat]: Does line 51 stay like this?

**Comment by efeyazgan** [comment]
May be you can follow https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideSubgroupMC#Measure_proper_value_of_qCut

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat (hunk 1 above)
Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat]: yes

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat [hunk 2]
```diff
@@ -256,7 +257,7 @@
 #*********************************************************************
 # Jet measure cuts                                                   *
 #*********************************************************************
- 0   = xqcut   ! minimum kt jet measure between partons
+ 20   = xqcut   ! minimum kt jet measure between partons
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat]: can you remind me the conclusion on the xqcut and qcut in pythia?

Comment (leonidasprado) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat]: Hello,
In a discussion with Aurore we decided to start with both xqcut and qcut as 20. We agreed that xqcut=0 was not good, but the value 20 is just a value we thought was reasonable ( I never used pythia before).

Comment (perrozzi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat]: I think xqcut=10 is appropriate and will give you some flexibility if needed

**Pull request merged**